client/i18n/locales/english/translations.json

```diff
@@ -9,6 +9,8 @@
     "frontend": "Front End",
     "backend": "Back End",
     "view": "View",
+    "view-code": "View Code",
+    "view-project": "View Project",
```

Comment:
```suggestion
    "view-code": "Code",
    "view-project": "Project",
```

Reply:
Makes sense, will change and add the preview icon as well.

Reply:
isn't this going to make localization harder?

Reply:
@ieahleen is correct these should be whole phrases, even at the cost of repeating ourselves. Assuming you can combine words is a bad idea. 

We should go back to adding individual keys for each label and keep it straight.

Reply:
I'll remove the previous commit and force push in that case.

Reply:
Thank you and sorry for the confusion